lecture_02/assignment_01/pedram_mirabian/assignment_01.py

```diff
@@ -0,0 +1,35 @@
+from compas.artists import Artist
+from compas.datastructures import Mesh
+from compas.geometry import Box
+from compas.geometry import Frame
+from compas.geometry import Plane
+from compas.geometry import Projection
+
+# Define a Frame, which is not in the origin and a bit tilted to the world frame
+point_1 = (1, 3, 5)
+xAxis_1 = (0.916893,0,-0.399133)
+yAxis_1 = (-0.04001,0.994963,-0.091911)
+
+frame = Frame(point_1, xAxis_1, yAxis_1)
+
+# Create a Box with that frame
+box = Box(frame,1,1.62,2.62)
```

Comment:
Code style: variables are usually all-lowercase (with some exceptions), so these one should be `xaxis_1` and `yaxis_1`. 
Also, always add a whitespace after commas:
```suggestion
xaxis_1 = (0.916893, 0, -0.399133)
yaxis_1 = (-0.04001, 0.994963, -0.091911)

frame = Frame(point_1, xaxis_1, yaxis_1)

# Create a Box with that frame
box = Box(frame, 1, 1.62, 2.62)
```